CONTRIBUTING.md

```diff
@@ -208,11 +208,24 @@ If you are a project maintainer, you can build and release a new version of
     ```ini
     # Your credentials for https://oss.sonatype.org/
     # NOTE: An equals sign (`=`) in any of these fields will break the parser
-    NEXUS_USERNAME=your-nexus-username
-    NEXUS_PASSWORD=your-nexus-password
+    # NOTE: Do not wrap any field in quotes
+
+    nexusUsername=your-nexus-username
```

Comment:
I had both `UPPER_SNAKE` and `camelCase` in my gradle.properties so I don't know which is necessary or if both are?

Reply:
Oh right! The mystery continues. I added both.